index.html

```diff
@@ -3,9 +3,217 @@
 <link rel="stylesheet" href="Styles/styles.css">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Webuildwebsites</title>
 </head>
 <body>
+<nav class="outer-container nav-section">
+    <div class="inner-container nav-inner">
+        <ul class="navbuttons">
+            <li>Case studies</li>
+            <li>Services</li>
+            <li>Contact us</li>
+        </ul>
+    </div>
+</nav>
+<header class="outer-container header-section">
+    <div class="inner-container header-sectionin">
+        <h1>Your website is an investment in the success of your business.</h1>
+        <h3>We are here to help you succeed.</h3>
+        <button>LET'S WORK TOGETHER</button>
+    </div>
+</header>
+<div class="outer-container header-img">
+    <div class="inner-container header-imgin">
+        <img class="imgline" src="assets/squiggle_line_stripe.svg">
+    </div>
+</div>
+<main>
+    <section class="outer-container section-1">
+        <div class="inner-container section-1in">
+            <div class="innerinner">
+                <p class="par1">Rekupe is a digital agency in Los Angeles.</p>
+                <p class="par2">We use strategic UX, SEO optimization and beautiful visual designs to help your business
+                    grow.</p>
+                <p class="par3">We see every project as a fresh opportunity, pushing ourselves to create new digital
+                    experiences.</p>
+            </div>
+        </div>
+    </section>
 
+    <div class="outer-container section-link">
+        <div class="inner-container section-linkin">
+            <p class="par4">Have a question?
+                <a href="...">Click here</a> to contact us!</p>
+        </div>
+    </div>
+
+
+    <section class="outer-container section-2">
+        <div class="inner-container section-2in">
```

Comment:
Je hebt nog veel divjes in divjes. Binnen een section kan je heel goed article gebruiken ter vervanging van een div.